firmware_configurations/klipper/TechnoPhreak/BTT_SKR_v2/printer.cfg

```diff
@@ -383,20 +379,17 @@ kick_start_time: 0.5
 ##	if your fan is not able to slow down effectively
 off_below: 0.10
 
-[heater_fan controller_fan]
+[controller_fan controller_fan]
 ##	Controller fan - XYE board, FAN2 Connector
 pin: PB5
+fan_speed: 0.3
 kick_start_time: 0.5
-heater: heater_bed

```

Comment:
Add this line back in for both controller fans. controller_fan works of both the steppers and the bed heater.

Reply:
done !

Reply:
I've added this as well for controller fans;
```
heater: heater_bed, extruder
stepper: stepper_x, stepper_y, stepper_y